client.js

```diff
@@ -359,7 +359,8 @@ write-data=${ path.resolve(config.instanceDirectory, instance) }\r\n
             '--rcon-port', Number(process.env.RCONPORT) || instconf.clientPort,
             '--rcon-password', instconf.clientPassword,
         ], {
-            'stdio': ['pipe', 'pipe', 'pipe']
+            'stdio': ['pipe', 'pipe', 'pipe'],
+			'detached': process.platform === "linux",
```

Comment:
Formatting issue?

Reply:
Nah, original code has spaces instead of tabs.  I suppose both lines could be changed to use tabs here.

Reply:
Most of the codebase uses tabs - just reformat to tabs the parts y'r touching, we should get it to uniform state :)